src/org-folding-and-outline-provider.ts

```diff
@@ -17,7 +17,7 @@ enum ChunkType {
     SECTION = SymbolKind.Constant,
     BLOCK = SymbolKind.Number
 }
-type Chunk = { type: ChunkType, title: string, level: number, startLine: number };
```

Comment:
Is it a recommended practice in TypeScript? 
Prefixing interface with "I" is not known as best practive in Java but I does not known for TS. Although this page <https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines> claims that is not TS guidelines but "_These are Coding Guidelines for Contributors to TypeScript_" but they suggest to "_Do not use "I" as a prefix for interface names."_. 

**Perhaps, we can embrace theses coding guidelines to have a clear guidelines reference?**

Reply:
About the I, this is a lint rule: `ERROR: src/org-folding-and-outline-provider.ts:20:11 - interface name must start with a capitalized I` if I don't do it.
It's then probably a rule of `tslint:latest`
But we can customize it.

(not that the guidelines you refer are about the core of typescript)

Reply:
Yes, I know the referred guidelines is for the core of typescript. If it is the default configuration of tslint, I'm ok to follow this practice.

Reply:
(I meant note rather than not :))

I'm open, still new to typescript hence not many preferences :)

Reply:
I well understood "note" :)
I'm in favor of keeping the default tslint configuration